src/service/verifier.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package helium.verifier;
+
+message file_info {
+  string key = 1;
+  file_type file_type = 2;
+  uint64 timestamp = 3;
+  uint64 size = 4;
+}
+
+message processed_files {
+  repeated file_info files = 1; 
+}
+
+message share {
+  bytes pub_key = 1;
+  bytes weight = 2;
+  uint64 timestamp = 3;
+  cell_type cell_type = 4;
+}
+
+message shares {
+  map<string, share> shares = 1;
+}
+
+message speed_share {
+  bytes pub_key = 1;
+  uint64 upload_speed = 2;
+  uint64 download_speed = 3;
+  uint32 latency = 4;
+  uint64 timestamp = 5;
+}
+
+message speed_share_list {
+  repeated speed_share speed_shares = 1;
+}
+
+message speed_shares {
+  map<string, speed_share_list> speed_shares = 1;
+}
+
+message moving_avg {
+  uint64 window_size = 1;
```

Comment:
A comment about units would helpful. Same for any other applicable fields in the new protos